Components/Overlay/src/OgreFont.cpp

```diff
@@ -528,6 +528,18 @@ namespace Ogre
                 // Advance a column
                 if(width)
                     l += (width + char_spacer);
+
+#ifndef HAVE_FREETYPE
+                if (buffer != NULL)
+                {
+                    int ix0, iy0, ix1, iy1;
+                    stbtt__bitmap gbm;
+                    stbtt_GetGlyphBitmapBoxSubpixel(&font, stbtt_FindGlyphIndex(&font, cp), scale, scale, 0.0f, 0.0f, &ix0, &iy0, &ix1, &iy1);
```

Comment:
why are you calling this?

Reply:
My bad, I missunderstood it, now I think it is okay